dev/tools/gen_localizations.dart

```diff
@@ -36,8 +38,14 @@ const String outputHeader = '''
 // @(regenerate)
 ''';
 
+/// Maps locales to resource key/value pairs.
```

Comment:
We're not generating docs for this app, so maybe '//' is appropriate

Reply:
The style guide encourages writing docs for private stuff too: https://www.dartlang.org/guides/language/effective-dart/documentation#consider-writing-doc-comments-for-private-apis